wiki/Reporting_bad_behaviour/es.md

```diff
@@ -2,7 +2,7 @@
 no_native_review: true
```

Comment:
Since we are changing this article, wouldn't it be best if we fully translate this as well to remove this flag?